protocols/relay/src/priv_client.rs

```diff
@@ -100,18 +104,22 @@ pub struct Client {
     queued_actions: VecDeque<Event>,
 }
 
-impl Client {
-    pub fn new_transport_and_behaviour(
-        local_peer_id: PeerId,
-    ) -> (transport::ClientTransport, Self) {
-        let (transport, from_transport) = transport::ClientTransport::new();
-        let behaviour = Client {
-            local_peer_id,
-            from_transport,
-            directly_connected_peers: Default::default(),
-            queued_actions: Default::default(),
-        };
-        (transport, behaviour)
+/// Create a new client relay [`Behaviour`] with it's corresponding [`Transport`].
+pub fn new(local_peer_id: PeerId) -> (transport::Transport, Behaviour) {
+    let (transport, from_transport) = transport::Transport::new();
```

Comment:
```suggestion
pub fn new(local_peer_id: PeerId) -> (Transport, Behaviour) {
    let (transport, from_transport) = Transport::new();
```

Don't think the qualification is necessary! :)

Reply:
and you are right, thanks! Updated